src/editors/codeedit.cpp

```diff
@@ -2853,5 +2853,29 @@ void CodeEdit::moveLines(bool moveLinesUp)
         mCompleter->suppressNextOpenTrigger();
 }
 
+void CodeEdit::trimtrailing()
+{
+    QTextCursor c= textCursor();
+    c.movePosition(c.Start);
```

Comment:
Instead of "c.Start" this should be "QTextCursor::Start"  ("c" is an instance, but statics like this enum should be referenced with the class itself)